subsys/bluetooth/host/att.c

```diff
@@ -2622,8 +2622,6 @@ static void att_reset(struct bt_att *att)
 		net_buf_unref(buf);
 	}
 
-	att->conn = NULL;
```

Comment:
The removal of `att->conn = NULL;` from this function needs the approval of @jhedberg or @Vudentz. I simply do not know if it is needed at all, or what consequences it may have.

Reply:
Do not set this to NULL